packages/rust-bindings/scripts/build-react-native-wallet.ts

```diff
@@ -1,9 +1,15 @@
 import path from 'node:path';
 
-import { convertWasmToJs, copyToFolder } from './build-scripts';
+import url from 'url';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+
+//import { convertWasmToJs, copyToFolder } from '../dist/scripts/build-scripts.js';
+import { copyToFolder } from '../dist/scripts/build-scripts.js';
+
 
 const walletBundlerPath = path.join(__dirname, '../lib/wallet/bundler');
 const walletOutPath = path.join(__dirname, '../lib/wallet/react-native');
 
 copyToFolder(walletBundlerPath, walletOutPath);
-convertWasmToJs(walletOutPath);
+//convertWasmToJs(walletOutPath);
```

Comment:
We cannot just remove this, as this step is needed specifically for the bindings to be compatible with react-native.